core/src/main/java/ai/timefold/solver/core/api/score/stream/common/IntervalBreak.java

```diff
@@ -11,21 +11,21 @@ public interface IntervalBreak<Interval_, Point_ extends Comparable<Point_>, Dif
     /**
      * @return never null, the interval cluster leading directly into this
      */
-    IntervalCluster<Interval_, Point_, Difference_> getPreviousIntervalCluster();
+    ConcurrentUsage<Interval_, Point_, Difference_> getPreviousConcurrentUsage();
 
     /**
      * @return never null, the interval cluster immediately following this
      */
-    IntervalCluster<Interval_, Point_, Difference_> getNextIntervalCluster();
+    ConcurrentUsage<Interval_, Point_, Difference_> getNextConcurrentUsage();
 
     /**
      * Return the end of the sequence before this break. For the
      * break between 6 and 10, this will return 6.
      *
      * @return never null, the item this break is directly after
      */
-    default Point_ getPreviousIntervalClusterEnd() {
-        return getPreviousIntervalCluster().getEnd();
+    default Point_ getPreviousConcurrentUsageEnd() {
```

Comment:
If we're changing the difference function to be `[start, end)`, we should consider changing this too.